01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,17 @@
+const http = require('http')
+
+const server = http.createServer((req, res) => {
+    if (req.url === '/'){
+    res.end('Welcome to our home page!')
+    }
+    if (req.url === '/about'){
+        res.end('Here is our short history')
+    }
+    res.end(`
+    <h1>Oops!</h1>
+    <p>We can't seem to find the page you're looking for</p>
+    <a href = "/" >back home</a>
+    `)
```

Comment:
I spy a bug! You should be able to replicate it by visiting the `/` or `/about` page. When you visit these pages, I'd expect the server to sometimes crash with an error message that says something like, "error: cannot write after stream end."

Can you try to replicate this error, then do some research and send me a stack overflow link that explains it's cause, then let me know how well you understand what's going on and whether you have an idea for how to fix it!

I also have 1:1 time slots tonight, so you can sign up if you want to work through this together.

Reply:
Oh wait, Diana, I'm not sure if I'm mixing up students, but we worked through this together in mentoring I think, right? 

In that case, you know how to fix it! Go ahead and add another commit to this branch with the fix!